scripts/install_dependencies/install_NEO_OCL_driver.sh

```diff
@@ -353,16 +244,16 @@ _uninstall_user_mode_ubuntu()
 
 uninstall_user_mode()
 {
-    if [[ $DISTRO == "centos" || $DISTRO == "redhat" ]]; then
-        _uninstall_user_mode_centos
+    if [[ $DISTRO == "redhat" ]]; then
+        _uninstall_user_mode_redhat
     else
         _uninstall_user_mode_ubuntu
     fi
 }
 
 _is_package_installed()
```

Comment:
Can we remove `_is_package_installed` function? Looks like it is unused.

Reply:
Agreed!